### PR TITLE
Added model comparision table

### PR DESCRIPTION
**Pull request by bdwyer2**
- Model size was calculated by checking the file size in my `~/.keras/models/` directory
- top-1 and top-5 accuracy was found on Google from a variety of sources
- Parameters was determined using the Keras `summary()` function
- Depth was determined using the Keras `layers_by_depth` function

image

### CODE REVIEW

**Comment by fchollet** [comment]
Great, looks good! Please add a sentence to specify that the top-1 and top-5 accuracy refers to the public ImageNet validation set.

**Comment by bdwyer2** [comment]
@fchollet thanks for merging this in!

I think this completes one of the tasks under the "Improvements" section on the [request for contributions page](https://github.com/fchollet/keras/projects/1).

**Comment by ozabluda** [comment]
Maybe it's a good idea to add "depth" to the model summary? Care needs to be taken to only include layers with "inherently" trainable parameters i.e. don't include pooling layers, but do include layers with layer.trainable==False. Maybe list both - trainable depth and total depth.
